gradle-plugins/rnbrownfield-gradle-plugin/example-android-library/build.gradle.kts

```
@@ -0,0 +1,47 @@
+
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("com.callstack.rnbrownfield-plugin")
+}
+
+rnbrownfield {}
```

Could we rename it to 
```suggestion
reactNativeBrownfield {}
```

---

BTW is having this, even as an empty block necessary?